src/utils/parseJsDoc.js

```diff
@@ -28,7 +29,14 @@ function getType(tag) {
   if (!tag.type) {
     return null;
   }
-  return {name: tag.type.name};
+  return {name: tag.type.name ? tag.type.name : tag.type.expression.name};
+}
+
+function getOptional(tag) {
+  if (tag.type && tag.type.type && tag.type.type === 'OptionalType') {
+    return true;
+  }
+  return;
```

Comment:
Ah, I think this should be `return false;`, but it's too late now anyway since I already published a new release :D